docs/apis/core/hooks/index.md

```diff
@@ -341,23 +409,19 @@ To make use of Stoppable events, the hook simply needs to implement the `Psr\Eve
 
 namespace core\hook;
 
+use core\attribute;
+
+#[attribute\label('A hook dispatched just before a block instance is deleted')]
+#[attribute\hook\replaces_callbacks('pre_block_delete')]
```

Comment:
This example is confusing because is following both approaches, implementing interface and attribute. I was expecting a similar reference about deprecated_callback_replacement interface vs replaces_callbacks attribute, as you did for the described_hook vs label/tags attributes :-)